src/index.ts

```diff
@@ -385,8 +389,14 @@ export async function cli(args: string[]) {
     namedExports: undefined,
     webDependencies: undefined,
   };
-  const doesWhitelistExist = !!webDependencies;
-  const arrayOfDeps = webDependencies || Object.keys(pkgManifest.dependencies || {});
+
+  let doesWhitelistExist = true;
+  const arrayOfDeps = [...webDependencies, ...scanImports(entry || '', {dest})];
```

Comment:
Also, can we use a `Set` to remove duplicates? 

- call `Array.from(new Set(arrayOfDeps))` at the end
- or, even use a Set throughout this function and then only convert to an array at the end